classes/command/class-duplicated-postmeta.php

```diff
@@ -64,7 +64,29 @@ public static function detect() {
 				WHERE meta_key IN (" . self::generate_placeholders( self::META_KEY_LIST, 1 ) . ")
 				GROUP by `post_id`
 				HAVING all_count <> unique_count 
-				ORDER BY `all_count` DESC ";
+				ORDER BY `all_count` DESC";
+
+		$prepared_sql = $wpdb->prepare( $sql, self::META_KEY_LIST );
+
+		return $wpdb->get_results( $prepared_sql );
+		// phpcs:enable
+	}
+
+	/**
+	 * Detect duplicate postmeta meta_key's with count
+	 *
+	 * @return object
+	 */
+	public static function detect_metakey_counts() {
```

Comment:
Can't this use the same function as for the admin page? Probably there it makes sense too to detect all meta keys.

Reply:
I updated the same logic for admin report and keep single detect method. Please have a look.